x/harvest/keeper/borrow.go

```diff
@@ -0,0 +1,138 @@
+package keeper
+
+import (
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/kava-labs/kava/x/harvest/types"
+)
+
+// USDX is the USDX coin's denom
+const USDX = "usdx"
+
+// Borrow funds
+func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, amount sdk.Coin) error {
```

Comment:
add keeper test for happy-path barrow

Reply:
Added in https://github.com/Kava-Labs/kava/pull/702/commits/25a9015bd845e9b260444d1578b5e928667a0031, https://github.com/Kava-Labs/kava/pull/702/commits/390116c0f1e3e480feb21ecf6257cc45b6e2a7c0